code/addons/onboarding/src/App.tsx

```diff
@@ -134,7 +142,12 @@ export default function App({ api }: { api: API }) {
           codeSnippets={codeSnippets}
           addonsStore={addons}
           onFinish={() => {
-            api.selectStory('example-button--warning');
+            try {
+              api.selectStory('example-button--primary');
```

Comment:
Shouldn't the selected story still be `warning`? And can you quickly elaborate, on why the safety catches are necessary or in which particular scenarios `selectStory` throws?

Reply:
I couldn't test it otherwise, because in the storybook I was testing it, this would break, as it didn't have the storyId.

Reply:
I'm happy to revert this, but I don't want to significantly change the code that's there. If we want to improve the addon that's a separate matter/PR?